Karen.java

```diff
@@ -0,0 +1,63 @@
+import com.sun.deploy.util.StringUtils;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Karen Xiao
+ * @see 5th Anniversary Class Report
+ * @since 11/9/18
+ */
+
+public class Karen
+{
+    private Integer age;
+    private String currentCity;
+    private String job;
+    private String company;
+    private static List<String> freeTime = Arrays.asList("Travel", "Dance", "Eat Hot Pot", "Paint", "Run", "Climb");
+
+    private Karen(Integer age, String currentCity, String job, String company)
+    {
+        this.age = age;
+        this.currentCity = currentCity;
+        this.job = job;
+        this.company = company;
+    }
+
+    private static void yearInTheLifeOf(Karen k, String hobby)
+    {
+        System.out.println(StringUtils.join(Arrays.asList(
```

Comment:
You could put the `toString` method on this `Karen` class and then make your main method look like this:
```
public static void main(String[] args)
{
    Karen karen = new Karen(20, "Cambridge, MA", "Student", "Harvard");
    karen.growUp();
}

//notice instance method
public void growUp()
{
    aYearInTheLifeOf(freeTime.get(0));
    
    for (int year = 1; year < 2; year++)
    {
         ...
    }
}

public void aYearInTheLifeOf(String hobby)
{
    System.out.println(this.toString() + " | " + hobby);
}

public void toString()
{
   //join thing here
}
```